### ISSUE TITLE
uMurmur for android 

### ISSUE THREAD

**Issue by ddamianus** [umurmur/umurmur#138]
Do you considering uMurmur for android?

**Comment by doctaweeks** [comment]
Can you elaborate on what you're trying to do? What is the real use case for this?

**Comment by concatime** [comment]
Someone closes this issue. uMurmur is a server, not a client, so irrelevant.

**Comment by fmorgner** [comment]
Closing since the issue is unrelated to uMurmur

**Comment by C4K3** [comment]
It's possible the OP was talking about the possibility of running a server on an old android phone, similar to how one runs servers on routers.

**Comment by Damianuscz** [comment]
Yep, but alsow on new android HW Phone/tablet. Sometimes i need portable server with proper configuration to run on local mumble server. uMurmur on android would be great alternative for local comunication.

**Comment by fdutheil** [comment]
I've thought of a nice use case for this, my fellow motorcycles riders that want "local" voice chat while on the road.

1. get _umurmur for android (c)_ on one android device
2. establish a local network topology of your choice (like BT or rather WIFI hotspot) amongst bikers mobile devices
3. join the local network and run a mumble client on each smartphone to join the _umurmur for android_ (c) instance.
4. enjoy 

Dedicated hardware for this use case is VERY expensive (hundreds of $/€ per kit) and each participant (biker or passenger) have to get one.

**Comment by Damianuscz** [comment]
Yep easy walkie talkie base on mumble

**Comment by tigrrrlily** [comment]
Apparently you just can't have a group voice solution for android that works locally on a hotspot. Does not exist, for love or money. SIP apps all miss one crucial feature or another... so yes, an android mumble server would be a godsend.

**Comment by Damianuscz** [comment]
It is sad solution i must take with me rpi zero with running server 😪

**Comment by troxor** [comment]
Let's consider this for a spin-off project, leaving this repo to be focused on the core uMurmur, staying true to "minimalistic" intention of this project.
